benchmarks/simplerouter/src/protocol/mod.rs

```diff
@@ -68,7 +68,7 @@ pub enum Error {
     InvalidReason(u8),
     #[error("Invalid protocol used")]
     InvalidProtocol,
-    #[error("Invalid protocol level")]
+    #[error("Invalid protocol level = {0}")]
```

Comment:
I think we missed noticing this `text:` vs `text =` thingy.

feel free to just change it on the default branch or create a PR yourself, that’s way quicker.

Reply:
Let's keep it like that